examples/with-react-i18next/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["next/babel"],
```

Comment:
@timneutkens will check later today and provide updates...thank you for reviewing...